Socials.yml

```diff
@@ -269,6 +287,9 @@ Snapchat:
 Sonico.com:
   - sonico.com
 
+SoundCloud:
+  - gate.sc
```

Comment:
Isn't gate.sc more like a marketing tool or redirection site? Wouldn't could that as social network. Also I don't think it's only used by SoundCloud, is it?

Reply:
Is used only by SoundCloud. The domain is registered to them: SoundCloud Global Limited & Co. KG